x-pack/plugins/rollup/public/test/client_integration/job_create_clone.test.js

```diff
@@ -29,11 +26,16 @@ describe('Cloning a rollup job through create job wizard', () => {
   let startMock;
 
   beforeAll(() => {
+    jest.useFakeTimers();
```

Comment:
I'm wondering why we would need `fakeTimers` if we don't use any of `advanceTimersByTime` or `runAllTimers`, would you mind explaining?

Reply:
We want to have the guarantee that there aren't any timeout running in the tests. We are not using jest for isolated unit tests but for component integration tests which load all the npm dependencies which could potentially use timeouts. And we don't want them in our test.

By adding this on top of our tests we are removing any risk of timing out so I would recommend to declare `jest.useFakeTimers();` to every component integration test file.